src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -120,24 +120,24 @@ public String getApiUri() {
     @Override public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {
         TaskListener listener = observer.getListener();
         if (repoOwner.isEmpty()) {
-            listener.getLogger().format("Must specify user or organization%n");
-            return;
+            throw new AbortException("Must specify user or organization%n");
```

Comment:
1. Create a Github Organization. Everything is indexed properly.
1. Edit the job configuration and remove the `Owner` (leave empty)
1. Save the Job (re-index)
1. All your jobs wil be removed

For that reason (by design) if something fails during the `computeChildren` we should throw an exception.